vignettes/sos4R.Rnw.old

```diff
@@ -1,1875 +0,0 @@
-\documentclass[nogin,a4paper]{article}
```

Comment:
What's the reason to remove this file? I agree it's not really helpful anymore, but wondering why as part of this PR.

Reply:
it is pure coincidence, no particular relation to this PR